migration/main.py

```diff
@@ -84,12 +89,39 @@ def main(api: API, account_id: int, term_ids: list[int], migrations: list[ToolMi
     account_id: int = int(os.getenv('ACCOUNT_ID', '0'))
     enrollment_term_ids: list[int] = convert_csv_to_int_list(os.getenv('ENROLLMENT_TERM_IDS', '0'))
 
+    wh_host = os.getenv('WH_HOST')
+    wh_port = os.getenv('WH_PORT')
+    wh_name = os.getenv('WH_NAME')
+    wh_user = os.getenv('WH_USER')
+    wh_password = os.getenv('WH_PASSWORD')
+
+    db: DB | None = None
+    if (
+        wh_host is not None and
+        wh_port is not None and
+        wh_name is not None and
+        wh_user is not None and
+        wh_password is not None
+    ):
```

Comment:
Is there a need for a warning message for missing warehouse configuration?

Reply:
I'll add a log message on the other path, not sure if it's a warning or just `INFO`.

Reply:
https://github.com/tl-its-umich-edu/tool-migration/pull/9/commits/48dd9021271688ff79a0dd362903d3fe26bc9981